wemake_python_styleguide/visitors/ast/exceptions.py

```diff
@@ -198,3 +200,21 @@ def _check_exception_type(self, node: ast.ExceptHandler) -> None:
         exception_id = getattr(exception_name, 'id', None)
         if exception_id == self._base_exception:
             self.add_violation(BaseExceptionViolation(node))
+
+    def _check_except_expression(self, node: ast.ExceptHandler) -> None:
+        # Catch-all 'except' is actually okay in this case
+        if node.type is None:
+            return
+
+        if isinstance(node.type, (ast.Name, ast.Attribute)):
+            return
+
+        if isinstance(node.type, ast.Tuple):
+            all_elements_are_trivial = all((
+                isinstance(element, (ast.Name, ast.Attribute))
```

Comment:
NIP: let's add some constant for trivial elements types, so it will be easier to extend in the future and thanks to this code repetitions (this tuple is also used in line 209) can be avoided

Reply:
👍